examples/Cliner/download_model.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+wget -P ../CliNER/models https://drive.google.com/file/d/1Jlm2wdmNA-GotTWF60zZRUs1MbnzYox2/view?usp=sharing
```

Comment:
Provide the current path and file name to save the model.

Reply:
Repleced it with the dowload_model.py.